test/safe/replace_test.rb

```diff
@@ -83,6 +83,9 @@ def test_replace_class
       e = assert_raises { House.new(:foo) }
       assert_equal "Unimplemented", e.message
     }.tap { |t| t.abort_on_exception = true }.join
+
+    error = assert_raises(NoMethodError) { House.color }
+    assert_match(/No method `ReplaceTest::House\.color'/, error.message)
```

Comment:
Would you mind adding part of the method stub (e.g. `def self.color`) in this assertion since that's the thing you were wanting to ensure was added to the error by making this change?